tests/gtest/.clang-format

```diff
@@ -0,0 +1,5 @@
+---
+Language: Cpp
+BasedOnStyle: Google
+DerivePointerAlignment: false
```

Comment:
The example in https://clang.llvm.org/docs/ClangFormatStyleOptions.html also sets `PointerAlignment` to `Left`:

```
Language: Cpp
# Force pointers to the type for C++.
DerivePointerAlignment: false
PointerAlignment: Left
```

Reply:
Done but I believe `PointerAlignment: Left` is set by `BasedOnStyle: Google` already.

Reply:
You are right. Would you like to remove `PointerAlignment: Left`?